syntaxes/ballerina.tmLanguage

```diff
@@ -5234,6 +5282,404 @@
+            <key>beginCaptures</key>
+            <dict>
+              <key>1</key>
+              <dict>
+                <key>name</key>
+                <string>keyword.other.unicode-value.regexp.ballerina</string>
+              </dict>
+              <key>2</key>
+              <dict>
+                <key>name</key>
+                <string>punctuation.other.unicode-value.begin.regexp.ballerina</string>
+              </dict>
+            </dict>
+            <key>end</key>
+            <string>(\})</string>
+            <key>endCaptures</key>
+            <dict>
+              <key>1</key>
+              <dict>
+                <key>name</key>
+                <string>punctuation.other.unicode-value.end.regexp.ballerina</string>
+              </dict>
+              <key>patterns</key>
+              <array>
+                <dict>
+                  <key>match</key>
+                  <string>([0-9A-Fa-f]{1,6})</string>
+                  <key>name</key>
+                  <string>constant.other.unicode-value.regexp.ballerina</string>
+                </dict>
+              </array>
+            </dict>
+          </dict>
+        </array>
+      </dict>
+      <key>flags-on-off</key>
+      <dict>
+        <key>name</key>
+        <string>meta.flags.regexp.ballerina</string>
+        <key>patterns</key>
+        <array>
+          <dict>
+            <key>begin</key>
+            <string>(\??)([imsx]*)(-?)([imsx]*)(:)</string>
+            <key>beginCaptures</key>
+            <dict>
+              <key>1</key>
+              <dict>
+                <key>name</key>
+                <string>punctuation.other.non-capturing-group-begin.regexp.ballerina</string>
+              </dict>
+              <key>2</key>
+              <dict>
+                <key>name</key>
+                <string>keyword.other.non-capturing-group.flags-on.regexp.ballerina</string>
+              </dict>
+              <key>3</key>
+              <dict>
+                <key>name</key>
+                <string>punctuation.other.non-capturing-group.off.regexp.ballerina</string>
+              </dict>
+              <key>4</key>
+              <dict>
+                <key>name</key>
+                <string>keyword.other.non-capturing-group.flags-off.regexp.ballerina</string>
+              </dict>
+              <key>5</key>
+              <dict>
+                <key>name</key>
+                <string>punctuation.other.non-capturing-group-end.regexp.ballerina</string>
+              </dict>
+            </dict>
+            <key>end</key>
+            <string>()</string>
+            <key>name</key>
+            <string>constant.other.flag.regexp.ballerina</string>
+            <key>patterns</key>
+            <array>
+              <dict>
+                <key>include</key>
+                <string>#regexp</string>
+              </dict>
+            </array>
+          </dict>
+        </array>
+      </dict>
     </dict>
   </dict>
 </plist>
```

Comment:
Missing the newline here

Reply:
This file is generated one.

Reply:
There should be a new line in here. Can you created a issue for this. A new line should be at the end while generating